public/js/chess.js

```diff
@@ -17,6 +17,13 @@ const checkMove = ( event ) => {
   }
 }
 
+const reset = () => {
+  manager = new GameManager()
+  renderBoard( manager.board )
+}
+
+
+//Rendering Functions
```

Comment:
This jumps out at me as a missing abstraction as well - maybe there's a `Board` object that we could create that could be responsible for containing the rendering functions.